app/lib/experiments.coffee

```diff
@@ -79,101 +72,45 @@ module.exports = class ExperimentServerClient
     @excludedReason = null
     @currentCohort = null
     @currentParticipant = null
-    Geordi.UserGetter.currentUserID = null
-
-  ###
-  when we first get participant, and the user has not started experiment in a previous sessions, we'll need to log it to Geordi
-  ###
-  checkForExperimentStartAndLogIt: (participant) ->
```

Comment:
this participant-tracking code was specific to the blanks experiment and is no longer needed.